types/basics.go

```diff
@@ -55,6 +59,9 @@ type MasterDerivationKey [masterDerivationKeyLenBytes]byte
 // Digest is a SHA512_256 hash
 type Digest [hashLenBytes]byte
 
+// Verifier is a state proof
+type Verifier [KeyStoreRootSize]byte
```

Comment:
Same comment about the name as https://github.com/algorand/java-algorand-sdk/pull/266#discussion_r740651941